tssc/step_implementers/package/__init__.py

```diff
@@ -1,5 +1,30 @@
-"""
-tssc.StepImplementers for the 'tag-source' TSSC step.
+"""tssc.StepImplementers for the 'tag-source' TSSC step.
+
+Step Configuration
+------------------
+All tssc.StepImplementers for this step should
+accept minimally the following configuration options.
+
+| Parameter       | Description
+|-----------------|------------
+| `TODO`          | TODO
+
+Results
+-------
+All tssc.StepImplementers for this step should
+minimally produce the following step results.
+
+| Result Key       | Description
+|------------------|------------
+| `artifacts`      | An array of dictionaries with information on the built artifacts.
+                     Minimally each dictionary entry should have a `path` key with the absolute path
+                     to the built artifact.
+
+**artifacts**
+
+| `artifacts`Key | Description
```

Comment:
@itewk Minor space issue -> 'artifacts'_Key

Reply:
i got tired of placing examples and didn't seem the as useful. if people want to add in some later we can. for now i think we will just do this as is.